docs/01/b.md

```diff
@@ -2,13 +2,13 @@
   [algebra]: https://github.com/non/algebra
   [Equivalence]: http://en.wikipedia.org/wiki/Equivalence_relation
 
-## Eq
+## `Eq`
```

Comment:
I think one of the reasons I didn't do this was how it looks as a header or on toc. Let me try out this PR locally.